### PR TITLE
fix: deleting all table column breaks markdown (close #215)

### LINKED ISSUE #215: Deleting all column remains bunch of '|'s
## Version

ALL

## Test Environment

Table Extension Disabled

## Current Behavior

image

- select all cols of a table
- right click
- select remove col
- remains empty table, tr
image

## Expected Behavior

remove table if all col is removed.

### PR DESCRIPTION
**Pull request by kyuwoo-choi**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description
#215 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/wysiwygCommands/tableRemoveCol.js [hunk 1]
```diff
@@ -23,34 +23,39 @@ const TableRemoveCol = CommandManager.command('wysiwyg', /** @lends RemoveCol */
...
+      if ($selectedCellsByManager.length < tbodyColLength) {
+        sq.saveUndoState(range);
+        let $nextFocus, $cell;
 
-        removeColByCell($cell);
-      }
+        if ($selectedCellsByManager.length > 1) {
+          const $tailCell = $selectedCellsByManager.last();
+          const $headCell = $selectedCellsByManager.first();
+          $nextFocus = $tailCell.next().length > 0 ? $tailCell.next() : $headCell.prev();
```
Comment (seonim-ryu) [inline src/js/wysiwygCommands/tableRemoveCol.js]: `$tailCell.next().length ?` 이렇게만 체크해도 되겠어요.

File: src/js/wysiwygCommands/tableRemoveCol.js [hunk 2]
```diff
@@ -23,34 +23,39 @@ const TableRemoveCol = CommandManager.command('wysiwyg', /** @lends RemoveCol */
...
-        const $headCell = $selectedCellsByManager.first();
-        $nextFocus = $tailCell.next().length > 0 ? $tailCell.next() : $headCell.prev();
 
-        removeMultipleColsByCells($selectedCellsByManager);
-      } else {
-        $cell = getCellByRange(range);
-        $nextFocus = $cell.next().length ? $cell.next() : $cell.prev();
+      if ($selectedCellsByManager.length < tbodyColLength) {
+        sq.saveUndoState(range);
+        let $nextFocus, $cell;
```
Comment (seonim-ryu) [inline src/js/wysiwygCommands/tableRemoveCol.js]: let 선언도 if문 안에서 해줄수 있지 않나요~?

**Comment by seonim-ryu** [comment]
[6/18] 리뷰 완료하였습니다.

File: src/js/wysiwygCommands/tableRemoveCol.js (hunk 1 above)
Comment (kyuwoo-choi) [reply src/js/wysiwygCommands/tableRemoveCol.js]: 옙!

File: src/js/wysiwygCommands/tableRemoveCol.js (hunk 2 above)
Comment (kyuwoo-choi) [reply src/js/wysiwygCommands/tableRemoveCol.js]: $cell을 안으로 옮겼습니다.

**Pull request merged**